cmd/logql-analyzer/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM golang:1.17.9 as build
+
+COPY . /src/loki
+WORKDIR /src/loki
+RUN make clean && CGO_ENABLED=0 go build ./cmd/logql-analyzer/
+
+FROM alpine:3.15.4
+
+RUN apk add --no-cache ca-certificates
+
+COPY --from=build /src/loki/logql-analyzer /usr/bin/logql-analyzer
+
+ENTRYPOINT [ "/usr/bin/logql-analyzer", "--server.http-listen-port=3001" ]
```

Comment:
Should we rather pull the port from a config file?

Reply:
we do not use the config files, just a few args to define the port and log level....anyway this port here is not needed, moved it to `docker-compose` file.